editoast/src/client/runserver.rs

```diff
@@ -54,6 +54,8 @@ pub struct RunserverArgs {
     /// The timeout to use when performing the healthcheck, in milliseconds
     #[clap(long, env = "EDITOAST_HEALTH_CHECK_TIMEOUT_MS", default_value_t = 1000)]
     health_check_timeout_ms: u64,
+    #[clap(long, env = "EDITOAST_TRAINS_TRAFFIC_PATH", default_value_t = String::from(""))]
+    trains_traffic_path: String,
```

Comment:
You should be able to use directly a `PathBuf` I believe. Look at `dynamic_assets_path` above.


```suggestion
    trains_traffic_path: PathBuf,
```